lib/ansible/modules/network/ovs/openvswitch_bridge.py

```diff
@@ -134,7 +134,8 @@ def map_obj_to_commands(want, have, module):
                 templatized_command = ("%(ovs-vsctl)s -t %(timeout)s"
                                        " br-set-external-id %(bridge)s")
                 command = templatized_command % module.params
-                if want['external_ids']:
+                if 'external_ids' in want and want['external_ids'] and\
```

Comment:
The condition can be simplified as 
`if want.get('external_ids') and have('external_ids')`

Reply:
Done